spec/features/user_edits_wishlist_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.feature "user edits wishlist" do
+  scenario "successfully" do
+    user = FactoryGirl.create(:user)
+    wishlist = FactoryGirl.create(:wishlist, user: user)
+    new_instrument = "ukelele"
+
+    visit edit_wishlist_path(wishlist, as: user)
+    fill_in "Artist", with: wishlist.artist
+    fill_in "Title", with: wishlist.title
+    fill_in "Instrument", with: new_instrument
+    click_button "Update Wishlist"
+
+    expect(page).to have_text "Instrument: #{new_instrument}"
+    expect(page).to have_text "Wishlist item updated successfully."
+  end
+
+  scenario "only if they have created it" do
```

Comment:
Personal preference of mine is to have this "only if they created it" handled in a controller spec rather than a feature spec. Following the "feature specs cover happy path" convention.